autoload/tagbar/types/uctags.vim

```diff
@@ -1297,7 +1297,8 @@ function! tagbar#types#uctags#init(supported_types) abort
         \ {'short' : 'T', 'long' : 'typedefs',    'fold' : 0, 'stl' : 0},
         \ {'short' : 'i', 'long' : 'interfaces',  'fold' : 0, 'stl' : 1},
         \ {'short' : 'c', 'long' : 'classes',     'fold' : 0, 'stl' : 1},
-        \ {'short' : 'e', 'long' : 'enumerators', 'fold' : 0, 'stl' : 0},
+        \ {'short' : 'E', 'long' : 'enumerators', 'fold' : 0, 'stl' : 0},
+        \ {'short' : 'A', 'long' : 'assertions',  'fold' : 0, 'stl' : 0},
```

Comment:
This change seems to be non-standard. I just pulled and recompiled the latest universal ctags and the `enumerators` kind is still a lower case `e`, and the `assertions` kind is not listed. Is this a custom definition you have somewhere? Or is it tied to a new change that is not yet merged into the latest stable universal ctags?

```
host ~$ ctags --version
Universal Ctags 6.1.0(eb42eec1b), Copyright (C) 2015-2023 Universal Ctags Team
Universal Ctags is derived from Exuberant Ctags.
Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
  Compiled: Jul 23 2024, 08:36:30
  URL: https://ctags.io/
  Output version: 0.0
  Optional compiled features: +wildcards, +regex, +gnulib_regex, +iconv, +option-directory, +xpath, +packcc, +optscript, +pcre2
host ~$ ctags --list-kinds=vera
c  classes
d  macro definitions
e  enumerators (values inside an enumeration)
f  function definitions
g  enumeration names
i  interfaces
l  local variables [off]
m  class, struct, and union members
p  programs
P  function prototypes [off]
s  signals
t  tasks
T  typedefs
v  variable definitions
x  external variable declarations [off]
h  included header files
D  cpp macro parameters [off]
host ~$
```